### ISSUE TITLE
Wiki "Adding ADIDNS Records" example not working

### ISSUE THREAD

**Issue by ldionmarcil** [Kevin-Robertson/Powermad#4]
I'm trying out the examples in the "[Adding ADIDNS Records](https://github.com/Kevin-Robertson/Powermad/wiki/Adding-ADIDNS-Records)" wiki and they are not working for me.

```powershell

PS C:\Users\Administrator>  New-ADIDNSNode -Node foo
[+] ADIDNS node foo added

PS C:\Users\Administrator> Enable-ADIDNSNode -Node foo
[+] foo enabled

PS C:\Users\Administrator>  Set-ADIDNSNodeAttribute -Node foo -DNSRecord (New-DNSRecordArray -DNSData 1.2.3.4)
ScriptHalted
At C:\Users\Administrator\Documents\adidns.ps1:3132 char:9
+         throw
+         ~~~~~
    + CategoryInfo          : OperationStopped: (:) [], RuntimeException
    + FullyQualifiedErrorId : ScriptHalted
```

Looking at [the documentation](https://github.com/Kevin-Robertson/Powermad/blob/master/Powermad.ps1#L3106) for `New-DNSRecordArray`, I see that the `DNSData` parameter is no longer used, yet is [still being referenced in the console output](https://github.com/Kevin-Robertson/Powermad/blob/master/Powermad.ps1#L3145-L3152). Legacy code, I guess? I suppose you're already aware of this and it's on the roadmap.

So I used the new parameters and tried again:

```powershell
PS C:\Users\Administrator> Set-ADIDNSNodeAttribute -node foo -Attribute dnsRecord -value (New-DNSRecordArray -Type A -Data 1.2.3.4)
[-] Exception calling "InvokeSet" with "2" argument(s): "Number of parameters specified does not match the expected number."
```

I found out that this is because the returned value for `New-DNSRecordArray` is `Object[]` but it really needs to be `Byte[]`. I was able to change the record manually with this cheap hack:

```powershell
PS C:\Users\Administrator> Set-ADIDNSNodeAttribute -node foo -Attribute dnsRecord -value ([Byte[]] (New-DNSRecordArray -Type A -Data 1.2.3.4))
[+] dnsRecord updated for foo
```

So basically, the return-type for `New-DNSRecordArray` probably needs to be Byte[] and the documentation for that function should be updated. I have zero PowerShell experience and don't feel confident enough to submit a pull request for this. I could try, however, if you don't have the time.

**Comment by Kevin-Robertson** [comment]
Thanks! I fixed the byte array issue and updated the 'DNSData' references. Yeah, that parameter was changed to just 'Data'.

**Comment by ldionmarcil** [comment]
Confirmed fixed! Thanks for the quick reply.

```powershell
PS C:\Users\Administrator> Set-ADIDNSNodeAttribute -node foo -Attribute dnsRecord -value (New-DNSRecordArray -Type A -Data 1.2.3.5)
[+] ADIDNS node foo dnsRecord attribute updated```
